lib/config/executor/container_vm.go

```diff
@@ -143,6 +143,9 @@ type ExecutorConfig struct {
 
 	// version
 	Version *version.Build `vic:"0.1" scope:"read-only" key:"version"`
+
+	// migrated - this is used during in memory migration to assign whether an execConfig is viable for a commit phase
+	Migrated bool
```

Comment:
This value is actually used by portlayer in memory only, so suppose that attribute should be added into container cache, instead of here.
This definition is only for the persistent configuration.